### PR TITLE
Parses NMR for ORCA (rebased)

### PR DESCRIPTION
**Pull request by berquist**
An updated version of #427.

Things remaining are:
- [x] clean up some of the text
- [x] make sure the test file is consistent with our other test files
- [x] parse the tensors for each atom into a dict with `paramagnetic`, `diamagnetic`, `total` labels instead of 3-element lists.

### CODE REVIEW

File: cclib/parser/data.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ class ccData:
         natom -- number of atoms (integer)
         nbasis -- number of basis functions (integer)
         nmo -- number of molecular orbitals (integer)
+        nmrtensors -- Nuclear magnetic resonance tensors (dict of dicts of array[2])
```
Comment (shivupa) [inline cclib/parser/data.py:69]: `Nuclear magnetic resonance shielding tensors (dict of dicts of array[2])` sounds a little better but this doesn't really matter. 

This dict of dicts is somewhat confusing. Perhaps adding a section in parsed data notes in the docs?

**Review by shivupa — APPROVED** [review]
Approved. Fleshing out some documentation about this would be nice.

**Comment by berquist** [comment]
Wait for docs.

**Review by shivupa — APPROVED** [review]
Thanks

**Pull request merged**